app/assets/stylesheets/_enhanced.scss

```diff
@@ -31,10 +31,10 @@
 @import 'layout/page_specific/all';
 
 // Components
-@import 'components/common/all';
 @import 'components/page_specific/all';
 @import 'components/dough_theme/variables';
 @import 'components/dough_theme/*/_all.scss';
+@import 'components/common/all';
```

Comment:
For this to work I had to move the component CSS import so they are loaded after the dough_theme variables, as there is a variable which is used by the newsletter button which is defined in dough